**Kualitas Udara Antar Negara Tahun 2023**

* **Kelompok: Kelompok 14**
* **Nama dan NRP:**
    1. [Dennis Immanuel Stevano Baskoro] - [5027261096]
    2. [Eric Fahrezi Hutagalung] - [5027261138]
    3. [Fikri Taqiyuddin Althaf]-[5027261077]
* **Topik:** Eksplorasi Data tentang Kualitas Udara Antar Negara Tahun 2023.

**Latar Belakang dan Pertanyaan**


Data ini menarik karena memperlihatkan kontras polusi dan cuaca di kota-kota lintas benua. Terlihat dinamika unik antarwilayah, seperti Rio de Janeiro yang memiliki PM2.5 dan O3 tinggi pada kelembapan rendah (29,30%), berbanding terbalik dengan Mumbai yang mencatatkan PM10 tertinggi pada kelembapan ekstrem (99,97%).


Pertanyaan: 
1. Berapa rata-rata, median, dan seberapa besar variasi (standar deviasi) tingkat PM2.5 di setiap kota?
2. Apakah kota dengan rata-rata kelembapan udara tinggi cenderung memiliki kadar PM10 dan NO2 yang lebih rendah dibanding kota bernilai kelembapan rendah?
3. Jenis polutan mana (PM2.5, PM10, NO2, SO2, CO, atau O3) yang memiliki rentang dan persebaran nilai paling bervariasi secara global?

In [11]:
import pandas as pd

df = pd.read_csv("global_air_quality_data_10000.csv")
print(df.shape)   # (jumlah baris, jumlah kolom)
df.head()

(10000, 12)


,City,Country,Date,PM2.5,PM10,NO2,SO2,CO,O3,Temperature,Humidity,Wind Speed
0,Bangkok,Thailand,2023-03-19,86.57,25.19,99.88,30.63,4.46,36.29,17.67,59.35,13.76
1,Istanbul,Turkey,2023-02-16,50.63,97.39,48.14,8.71,3.40,144.16,3.46,67.51,6.36
2,Rio de Janeiro,Brazil,2023-11-13,130.21,57.22,98.51,9.92,0.12,179.31,25.29,29.30,12.87
3,Mumbai,India,2023-03-16,119.70,130.52,10.96,33.03,7.74,38.65,23.15,99.97,7.71
4,Paris,France,2023-04-04,55.20,36.62,76.85,21.85,2.00,67.09,16.02,90.28,14.16


**Deskripsi Data**

* Sumber data dan lisensi: (https://www.kaggle.com/datasets/waqi786/global-air-quality-dataset), dengan lisensi Apache 2.0.
* Pengumpulan data: oleh Waqar Ali, data kualitas udara ini dikumpulkan dari beberapa stasiun pemantauan resmi yang berlokasi di masing-masing kota. Pengukuran dicatat secara berkala dan dikompilasi untuk membentuk himpunan data (dataset) komprehensif yang mencakup berbagai polutan serta faktor lingkungan. Data tersebut kemudian dibersihkan dan diformat untuk memastikan siap digunakan untuk analisis. Data tersebut dikumpulkan dari berbagai sumber terpercaya, termasuk lembaga lingkungan hidup pemerintah dan organisasi internasional yang berfokus pada pemantauan kualitas udara. Data ini telah diagregasi dan diproses untuk memastikan konsistensi serta akurasinya. 
* Jumlah baris dan kolom: 10.000 baris dan 12 kolom.

**Kamus Data**

| Kolom | Arti | Jenis variabel | Skala | Satuan | Contoh |
| --- | --- | --- | --- | --- | --- |
| **City** | Nama kota lokasi pengukuran | Kualitatif | Nominal | - | Bangkok |
| **Country** | Nama negara lokasi kota berada | Kualitatif | Nominal | - | Thailand |
| **Date** | Tanggal waktu perekaman data | Kuantitatif kontinu | Interval | YYYY-MM-DD | 2023-03-19 |
| **PM2.5** | Konsentrasi partikel halus (<= 2.5 µm) di udara | Kuantitatif kontinu | Rasio | µg/m³ | 86.57 |
| **PM10** | Konsentrasi partikel kasar (<= 10 µm) di udara | Kuantitatif kontinu | Rasio | µg/m³ | 25.19 |
| **NO2** | Konsentrasi gas Nitrogen Dioksida di udara | Kuantitatif kontinu | Rasio | µg/m³ | 99.88 |
| **SO2** | Konsentrasi gas Sulfur Dioksida di udara | Kuantitatif kontinu | Rasio | µg/m³ | 30.63 |
| **CO** | Konsentrasi gas Karbon Monoksida di udara | Kuantitatif kontinu | Rasio | mg/m³ | 4.46 |
| **O3** | Konsentrasi gas Ozon Permukaan di udara | Kuantitatif kontinu | Rasio | µg/m³ | 36.29 |
| **Temperature** | Suhu udara saat pengukuran | Kuantitatif kontinu | Interval | °C | 17.67 |
| **Humidity** | Kelembapan relatif udara saat pengukuran | Kuantitatif kontinu | Rasio | % | 59.35 |
| **Wind Speed** | Kecepatan angin saat pengukuran | Kuantitatif kontinu | Rasio | m/s | 13.76 |